### PR TITLE
Update VMWare installation guide

### LINKED ISSUE #533: Update the Clear Linux VMWare installation guide
Update the content of the https://clearlinux.org/documentation/clear-linux/get-started/virtual-machine-install/vmw-player documentation to use the correct image to create a VM

### PR DESCRIPTION
**Pull request by bd-dean**
Closes #533 

Questions for review: 
1. we mention that we have a guide for download-verify-decompress on MacOS, but VMWare player doesn't run on MacOS.  Should this be removed? 
2. I don't have a windows system to recreate changed screenshots (figure 3, figure5) so I added verbiage explaining the changes. Please verify these are ok.

### CODE REVIEW

File: source/clear-linux/get-started/virtual-machine-install/vmw-player.rst [hunk 1]
```diff
@@ -66,7 +66,7 @@ Download the latest |CL| installer ISO
 **************************************
 
 Get the latest |CL| installer ISO image from the `image`_ repository.
```
Comment (puneetse) [inline source/clear-linux/get-started/virtual-machine-install/vmw-player.rst]: Let's point them to https://clearlinux.org/downloads instead now that it exists

File: source/clear-linux/get-started/virtual-machine-install/vmw-player.rst [hunk 2]
```diff
@@ -140,12 +140,12 @@ Create and configure a new VM
...
       :scale: 100%
-      :alt: VMware Workstation 14 Player - Set disk capacity
+      :alt: VMware Workstation Player - Set disk capacity
 
-      Figure 5: VMware Workstation 14 Player - Set disk capacity
+      Figure 5: VMware Workstation Player - Set disk capacity
 
    .. note::
-      A minimal |CL| installation can exist on 600MB of drive space.
+      A minimal |CL| installation can exist on 600MB of drive space. For optimal performance with the |CL| Desktop image, we recommend 32GB of drive space.
```
Comment (puneetse) [inline source/clear-linux/get-started/virtual-machine-install/vmw-player.rst]:
```suggestion
      For optimal performance with the |CL| Desktop image, we recommend 32GB of drive space.
```

While true in theory, this is actually not true anymore with our installers unfortunately. The ref to system-requirements is good and can provide more details for people looking for a minimal install

File: source/clear-linux/get-started/virtual-machine-install/vmw-player.rst [hunk 3]
```diff
@@ -114,13 +114,13 @@ Create and configure a new VM
...
-      :alt: VMware Workstation 14 Player - Select guest operating system type
+      :alt: VMware Workstation Player - Select guest operating system type
 
-      Figure 3: VMware Workstation 14 Player - Select guest operating system
+      Figure 3: VMware Workstation Player - Select guest operating system
       type
 
 #. Set the :guilabel:`Version` setting to
-   :guilabel:`Other Linux 3.x or later kernel 64-bit`.
+   :guilabel:`Other Linux x.x or later kernel 64-bit`.
```
Comment (puneetse) [inline source/clear-linux/get-started/virtual-machine-install/vmw-player.rst]:
```suggestion
   :guilabel:`Other Linux 4.x or later kernel 64-bit`.
```

x.x is too generic in this case becasue the drop down has similarly named options for 3x, 2x, etc so 4.x should be mentioned specifically.

x.x is too generic unfortunately, bec

**Comment by puneetse** [comment]
@bd-dean I can help refresh the screenshot in this PR, if you don't have someone else in mind to help already.

File: source/clear-linux/get-started/virtual-machine-install/vmw-player.rst (hunk 1 above)
Comment (bd-dean) [reply source/clear-linux/get-started/virtual-machine-install/vmw-player.rst]: good call, updated with latest commit.

File: source/clear-linux/get-started/virtual-machine-install/vmw-player.rst (hunk 2 above)
Comment (bd-dean) [reply source/clear-linux/get-started/virtual-machine-install/vmw-player.rst]: Removed the 600MB, though I may take it as a personal challenge :)

File: source/clear-linux/get-started/virtual-machine-install/vmw-player.rst (hunk 3 above)
Comment (bd-dean) [reply source/clear-linux/get-started/virtual-machine-install/vmw-player.rst]: Changed back to 4.x.  @puneetse if you can update the image for this (vmw-player-03.png) that would be great.

**Comment by bd-dean** [comment]
> @bd-dean I can help refresh the screenshot in this PR, if you don't have someone else in mind to help already.

There's nobody else in mind -- tag you're it!  Thanks @puneetse !

File: source/clear-linux/get-started/virtual-machine-install/vmw-player.rst [hunk 4]
```diff
@@ -65,18 +65,18 @@ For additional help, see the `VMware Workstation Player Documentation`_.
 Download the latest |CL| installer ISO
 **************************************
 
-Get the latest |CL| installer ISO image from the `image`_ repository.
-Look for :file:`clear-[version number]-installer.iso.xz`.
+Get the latest |CL| installer ISO image from the `download`_ repository.
+Look for :file:`clear-[version number]-live-desktop.iso.xz`.
```
Comment (mvincerx) [inline source/clear-linux/get-started/virtual-machine-install/vmw-player.rst]: Please simplify this. Something like: 
"Visit our downloads page to get the latest installer with Clear Linux OS Desktop." 
(The previous version is a holdover from when we didn't have the Downloads page.) 
I presume you prefer to use the GUI installer here.

**Comment by mvincerx** [comment]
@bd-dean. Let me know when you're close. I can help resolve the merge conflict if you want. 
I didn't see a new screenshot in the PR yet.

**Comment by puneetse** [comment]
I have pushed an update with refreshed screenshots and refactored the UEFI boot section

Another thing that should be added to this doc is installing the open-vm-tools. (`sudo swupd bundle-add os-cloudguest-vmware` and `systemctl enable --now open-vm-tools`.

File: source/clear-linux/get-started/virtual-machine-install/vmw-player.rst (hunk 4 above)
Comment (bd-dean) [reply source/clear-linux/get-started/virtual-machine-install/vmw-player.rst]: Updated with latest commit.

**Comment by bd-dean** [comment]
I am finished, I believe.  I will need help with the merge conflict though...thanks for offering @mvincerx !

**Comment by mvincerx** [comment]
np. @bd-dean, do you want to incorporate @puneetse 's comments re `open-vm-tools`? 
I'll wait to resolve the conflicts until after you two work this out.

**Comment by mvincerx** [comment]
Great. Resolving conflicts now.

**Comment by mvincerx** [comment]
@bd-dean.  Ready for your final review. I've now resolved all merge conflicts. Be sure to review this commit: 
https://github.com/clearlinux/clear-linux-documentation/blob/cba448ef2645ce8861fbcd7c7c437916ca81c2ee/source/clear-linux/get-started/virtual-machine-install/vmw-player.rst#id147

**Comment by bd-dean** [comment]
Looks great -- all updates added!

**Comment by mvincerx** [comment]
Thanks all.

**Pull request merged**